Source/Schema.NET/ValuesJsonConverter.cs

```diff
@@ -212,6 +161,216 @@ public override void WriteJson(JsonWriter writer, object value, JsonSerializer s
         public virtual void WriteObject(JsonWriter writer, object value, JsonSerializer serializer) =>
             serializer.Serialize(writer, value);
 
+        private static object ParseTokenArguments(JToken token, JsonToken tokenType, Type type, object value)
+        {
+            const int SCHEMA_ORG_LENGTH = 18;

```

Comment:
Can we please add a comment about what this length is?

Reply:
Sure, I just flattened that "inline string".Length into a constant.